Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-settings/test-launch-dialog.component.ts

```diff
@@ -1,8 +1,18 @@
-﻿import { Component } from "@angular/core";
+﻿import { Component } from '@angular/core';
+import { Test } from '../tests.model';
 
 @Component({
     moduleId: module.id,
     selector: 'test-launch-dialog',
-    templateUrl: "test-launch-dialog.html"
+    templateUrl: 'test-launch-dialog.html'
 })
-export class TestLaunchDialogComponent { }
+export class TestLaunchDialogComponent {
+    copiedContent: boolean;
+    settingObject: Test;
+
+    constructor() {
+        this.settingObject = new Test();
```

Comment:
again still name has only settings, instead of testSettings. Why there is a lack of attention towards the comments we are giving. We are spending time from our work to review PR, and for each line, we have to give updates. Is it not your responsibility to change similar issues throughout the files you have changed?